types/openui5/sap.uxap.d.ts

```diff
@@ -7658,7 +7648,7 @@ declare module "sap/uxap/ObjectPageSectionBase" {
      *
      * @ui5-protected Do not call from applications (only from related classes in the framework)
      */
-    setTitleVisible(bVisible: boolean): void;
+    setTitleVisible(): void;
```

Comment:
This looks suspicious at a first glance, but the implementation indeed ignores any parameters.